controllers/PasswordController.js

```diff
@@ -61,22 +65,28 @@ class PasswordResetController {
     const { users } = UserModel;
     const { id } = req.decoded;
     const hashedPassword = bcrypt.hashSync(req.body.password, 8);
+
     try {
       const user = await users.findOne({
         where: { id }
       });
+
       if (!user) {
         return Response.notfound(res, { message: 'user not avalaible' });
       }
-      const updatedPassword = await users.update({
-        where: {
-          id
-        },
-        password: hashedPassword
+      if (!user.resettingPassword) {
+        return Response.badRequest(res, {
+          message: 'this link has already been used to reset your password'
+        });
+      }
+      // update password
+      const updatedPassword = await user.update({
+        password: hashedPassword,
+        resettingPassword: false
       });
       if (updatedPassword) {
         return Response.success(res, {
-          message: 'password update was succesful'
+          message: 'password update was succesfull'
```

Comment:
succesfull has one `l` and two `s` - `successful`

Reply:
done